{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/models.py

```diff
@@ -5,10 +5,17 @@
 
 
 class User(AbstractUser):
+    """Default user for {{cookiecutter.project_name}}.
+    """
 
-    # First Name and Last Name do not cover name patterns
-    # around the globe.
+    #: First and last name do not cover name patterns around the globe
     name = CharField(_("Name of User"), blank=True, max_length=255)
 
     def get_absolute_url(self):
+        """Get url for user's detail view.
+
+        Returns:
+            str: URL for user detail.
+
+        """
```

Comment:
Use google style docstring, picked up by apidoc, as an example.
The comments are unneeded, but it shows how this works.